template-profile.php

```diff
@@ -8,13 +8,12 @@
 	$html_file = wp_get_attachment_url( $html_id );
 
 	// Set a timeout
-	$opts = array('http' => array(
-						'method'  => 'GET',
-						'timeout' => FEED_FETCH_TIMEOUT
-	));
+	$args = array(
+		'timeout' => FEED_FETCH_TIMEOUT
+	);
 	$context = stream_context_create($opts);
```

Comment:
This line can be removed

Reply:
The line needed to stay, but the variable needed to be updated.